src/app/data/forms/form.ts

```diff
@@ -1,23 +1,23 @@
 import { Control } from './control';
 import { XmlClass } from '../../decorators/xml.decorators';
+import { Container } from './container';
 
 @XmlClass('form')
-export class Form {
-  title = '';
+export class Form extends Container {
```

Comment:
Das ist Absicht, dass das kein Container oder Control ist. Es soll keiner auf den Gedanken kommen Form als Control zu benutzen und Forms in Forms zu schachteln.